meson.build

```diff
@@ -42,7 +42,13 @@ if with_xf86gamma_ext
 endif
 
 c = meson.get_compiler('c')
-dep_pam = c.find_library('pam', required: true)
+
+with_bsd_auth = get_option('with-bsd_auth')
+
+if with_bsd_auth == false
+  # BSDs don't have linux PAM
+  dep_pam = c.find_library('pam', required: true)
```

Comment:
Does BSD Auth have an associated pkgconfig that needs to be present? If so, that should be checked instead of dep_pam if the option is set to true, rather than avoiding any checks whatsoever.

Reply:
I didn't find a .pc files on /usr/lib or /usr/local/lib on OpenBSD... Do we need to check with header file ?

Reply:
If there isn't an associated .pc file, then it's fine to omit.